src/documentation/js/index.js

```diff
@@ -79,138 +233,6 @@ $(".modal .close").click(function () {
-  $(this).on("click", function (e) {
-    e.preventDefault();
-    var $index = index + 1;
-    $(".tab--item a, .tab--content").removeClass("active");
-    $(this).closest(".nav-tabs").attr("data-tab-active", $index);
-    $(this).addClass("active");
-    $(".tab--content:nth-child(" + $index + ")").addClass("active");
-    $(".tab--content .dp-plan-detail").slideUp();
-  });
-});
-
-// module accordion
-$("body").on("click", ".dp-accordion .dp-accordion-thumb", function (e) {
-  e.preventDefault();
-  $(this).parent().toggleClass("active");
-  $(this).parent().find(".dp-accordion-panel").slideToggle();
-});
-
-// Doppler plus - Plans details
-$(".dp-compare-details-plans").on("click", function (e) {
-  e.preventDefault();
-  $(this).toggleClass("dp-open-compare");
-  $(".dp-plan-detail").slideToggle();
-});
-
-// Slider sample one
-$(".progress-bar").each(function (index) {
-  $max = $(this).find(".range-slider").attr("max");
-  $(this)
-    .find(".range-slider")
-    .on("change input", function () {
-      var $quote = $(this).val(),
-        $progress = ($quote * 100) / $max,
-        $parent = $(this).parent();
-
-      $parent.find(".progress-anchor").css("width", $progress + "%");
-      $(this).attr("data-value", this.value);
-    })
-    .trigger("change");
-});
-
-// Content | Custom field | emojis
-$(".dp-bd-box .dp-button-bd").on("click", function () {
-  $(".dp-content-bd").hide();
-  $(".dp-button-bd").removeClass("active");
-  $(this).addClass("active");
-  $(this).parent().find(".dp-content-bd").show();
-});
-
-$(document).on("click", function (e) {
-  var container = $(".dp-content-bd"),
-    button = $(".dp-button-bd");
-
-  if (
-    !$(".dp-content-bd").is(e.target) &&
-    $(".dp-content-bd").has(e.target).length === 0 &&
-    !$(".dp-button-bd").is(e.target) &&
-    $(".dp-button-bd").has(e.target).length === 0
-  ) {
-    container.hide();
-    $(".dp-button-bd").removeClass("active");
-  }
-});
-
-// Button Exit editor
-$(".dp-button-box .dp-button-exit").on("click", function () {
-  $(this).toggleClass("active");
-  if ($(this).hasClass("active")) {
-    $(this).parent().find(".dp-content-menu").show();
-  } else {
-    $(this).parent().find(".dp-content-menu").hide();
-  }
-});
-
-$(document).on("click", function (e) {
-  var container = $(".dp-content-menu"),
-    button = $(".dp-button-exit");
-
-  if (
-    !$(".dp-content-menu").is(e.target) &&
-    $(".dp-content-menu").has(e.target).length === 0 &&
-    !$(".dp-button-exit").is(e.target) &&
-    $(".dp-button-exit").has(e.target).length === 0
-  ) {
-    container.hide();
-    $(".dp-button-exit").removeClass("active");
-  }
-});
-
-/* nested table | expand results */
-
-$(".dp-expand-results").on("click", function (e) {
-  e.preventDefault();
-  $(this).toggleClass("dp-open-results");
-  $(this).closest("tr").next().toggleClass("show");
-});
-
 // for all functions use dopplerUI namespace
 var dopplerUI = dopplerUI || {};
```

Comment:
Esto tiene pinta de estar acá para utilizar en los clientes. Que opinás de modificarlo de la siguiente manera?

```suggestion
window.dopplerUI = window.dopplerUI || {};
```

Reply:
Si eso esta dando un error en la documentación, me gusta la sugerencia.